packages/fullstack-hooks/src/hooks/server_cached.rs

```diff
@@ -2,9 +2,12 @@ use dioxus_core::use_hook;
 use dioxus_fullstack_protocol::SerializeContextEntry;
 use serde::{de::DeserializeOwned, Serialize};
 
-/// This allows you to send data from the server to the client. The data is serialized into the HTML on the server and hydrated on the client.
+/// This allows you to send data from the server to hydrate the client.
+/// - When compiled as server, the closure is ran and the resulting data is serialized on the server and sent to the client.
+/// - When compiled as web client, the data is deserialized from the server.
```

Comment:
The closure can also be run on the client if the hook is run for the first time on the client. For example, if you have a component that calls a endpoint inside of `use_server_cached` on a certain route, then the hook will be run on the client if you navigate to the route on the client

Reply:
Much more clear